### PR TITLE
fix(style): Object Style blocked new relationships

### PR DESCRIPTION
**Pull request by Balcan**
[jira issue](https://jira.dhis2.org/browse/ANDROAPP-2507)

### CODE REVIEW

**Comment by Balcan** [comment]
Before reviewing this issue we have to merge the previous ones.

**Comment by mmmateos** [comment]
Tested :ok_hand:

File: app/src/main/java/org/dhis2/usescases/teiDashboard/DashboardRepositoryImpl.java [hunk 1]
```diff
@@ -242,20 +243,16 @@ public Event updateState( Event eventModel, EventStatus newStatus )
     @Override
     public Integer getObjectStyle( Context context, String uid )
     {
-        String GET_OBJECT_STYLE = "SELECT * FROM ObjectStyle WHERE uid = ?";
-        try (Cursor objectStyleCurosr = briteDatabase.query( GET_OBJECT_STYLE, uid ))
+        TrackedEntityType teType = d2.trackedEntityModule().trackedEntityTypes().uid( uid ).blockingGet();
```
Comment (JaimeToca) [inline app/src/main/java/org/dhis2/usescases/teiDashboard/DashboardRepositoryImpl.java]: can teType be null?

Comment (Balcan) [reply app/src/main/java/org/dhis2/usescases/teiDashboard/DashboardRepositoryImpl.java]: No it can't :)

**Pull request merged**